src/database/models.py

```diff
@@ -1747,48 +1747,43 @@ class Meta:
         db_table = "carbon_equivalencies"
 
 
-class Vendor(models.Model):
+class Affiliate(models.Model):
```

Comment:
I suggest instead of Affiliate (which implies some affiliation, even its just a name) to call the base class Org